deps/bootstrapper/Dockerfile

```diff
@@ -127,7 +127,11 @@ RUN npm install -g --unsafe-perm ganache@7.9.0
 
 WORKDIR /app
 
-RUN git clone --recurse-submodules https://github.com/madara-alliance/madara-bootstrapper.git .
+ENV BOOTSTRAPPER_VER="a99088a96a8126d33fef4fabba76a70032e601cf"
```

Comment:
not sure why this is needed since bootstrapper is in the monorepo now, but maybe add a comment from where this value comes from to give more context

Reply:
Changed to main branch now that the 3 projects are there